packages/plugin-typescript/sources/typescriptUtils.ts

```diff
@@ -0,0 +1,31 @@
+import {Configuration, Descriptor} from '@yarnpkg/core';
+import {httpUtils, structUtils}    from '@yarnpkg/core';
+
+// Note that the appId and appKey are specific to Yarn's website - please
```

Comment:
👋 @Haroenv, can we use our credentials here? This would be used when `plugin-typescript` is enabled (disabled by default) and a dependency gets added via `yarn add`.

>This plugin automatically adds `@types/` packages into your dependencies when you add a package that doesn't include its own types.

Reply:
I’ll make a new api key for this tomorrow.

Reply:
`e8e1bd300d860104bb8c58453ffa1eb4`